go.mod

```diff
@@ -31,3 +32,5 @@ require (
 	google.golang.org/protobuf v1.36.2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/elastic/opentelemetry-lib/elasticattributes => ./elasticattributes
```

Comment:
IIUC bear in mind that replace is only for local development, and will not be respected by users of opentelemetry-lib. i.e. let's say apm-server uses opentelemetry-lib, it will need to pin to both the version of opentelemetry-lib and elasticattr.

We'll need to ensure this plays well in our use cases.

Reply:
I updated the PR for this part. I [pushed a tag for this new module](https://github.com/elastic/opentelemetry-lib/releases/tag/elasticattr%2Fv0.1.0) (originally created the tag to just show it in our slack discussion) and used that here. With that, there is no need for replace anymore.

That's what I meant in above comment.

Reply:
Update: since we move on as a package, we won't need this part. Already reverted.